OpenTabletDriver.Configurations/Configurations/UGEE/U1200.json

```
@@ -0,0 +1,37 @@
+{
+  "Name": "UGEE U1200",
+  "Specifications": {
+    "Digitizer": {
+      "Width": 263.5,
+      "Height": 147.5,
```

If using the digitizer's sensor resolution, the width and height should be

```suggestion
      "Width": 263.22,
      "Height": 148.055,
```
> `expected size = sensor res / lpi` where `lpi` is 200

The width is most likely more accurate and should change to it, but can you confirm if this suggested height is closer to reality? Instead of measuring the marked active area on the surface of the tablet, try measuring the bounds where the digitizer says it's 0 and whatever is the maximum for that axis.

When in doubt or can't measure with that much accuracy, just use this suggested width and height.

---

Yes, those values do appear to be more accurate when measuring that way. Thanks for the suggestion.